### PR TITLE
the calendar diplayed

### PR DESCRIPTION
**Pull request by roman-zherebetskyi**
- [DEMO LINK](https://roman-zherebetskyi.github.io/layout_calendar/)

### CODE REVIEW

File: src/styles/blocks/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    }
+
+    @for $number from 1 through 31 {
+      &:nth-child(#{$number})::before {
+        content: "#{$number}";
+      }
+    }
+
+    @each $day, $day-index in $days {
+      .calendar--start-day--#{$day} &:first-child {
```
Comment (igor-1987) [inline src/styles/blocks/calendar.scss]: Use "&" here, like &--start-day--#{$day}

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        margin-left: ($cell-size + $gap) * $day-index;
+      }
+    }
+
+  }
+
+  @for $day from 28 through 31 {
+    &--month-length--#{$day} {
+      .calendar__day:nth-child(n + #{$day + 1}) {
```
Comment (igor-1987) [inline src/styles/blocks/calendar.scss]: And here you can do it as well without using ".calendar" in selector directly.

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  perspective: $cell-size;
+
+  &__day {
+    box-sizing: border-box;
+    @include cell($cell-size);
+    border: 1px solid black;
+    background-color: #eee;
+    text-align: center;
+    line-height: $cell-size;
+    transition: $transition-time;
```
Comment (igor-1987) [inline src/styles/blocks/calendar.scss]: To leave "transition-property" blank is the same as to pass there "all", which is not OK. It's better to list them like "background $transition-time, transform $transition-time".

**Review by igor-1987 — COMMENTED** [review]
Nice job, but there are couple tiny details which can be improved.

File: src/styles/blocks/calendar.scss [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  gap: $gap;
+  padding: $paddings;
+  max-width: $calendar-width;
+  perspective: $cell-size;
+
+  &__day {
+    box-sizing: border-box;
+    @include cell($cell-size);
+    border: 1px solid black;
+    background-color: #eee;
```
Comment (igor-1987) [inline src/styles/blocks/calendar.scss]: It's better to keep colors in variables.

File: src/styles/blocks/calendar.scss (hunk 1 above)
Comment (roman-zherebetskyi) [reply src/styles/blocks/calendar.scss]: got it!

File: src/styles/blocks/calendar.scss (hunk 2 above)
Comment (roman-zherebetskyi) [reply src/styles/blocks/calendar.scss]: understood!

File: src/styles/blocks/calendar.scss (hunk 3 above)
Comment (MaksimusCRS) [reply src/styles/blocks/calendar.scss]: Fix it

**Review by MaksimusCRS — CHANGES_REQUESTED** [review]
Fix all student's comments before review

File: src/styles/blocks/calendar.scss (hunk 3 above)
Comment (roman-zherebetskyi) [reply src/styles/blocks/calendar.scss]: fixed

**Review by MaksimusCRS — APPROVED** [review]
GJ

**Pull request closed without merging**